vic/drivers/image/src/vic_force.c

```diff
@@ -220,6 +321,17 @@ vic_force(void)
                                                 param.SNOW_MAX_SNOW_TEMP,
                                                 &(atmos[i].prec[j]), 1);
         }
+        // Validate vegcover
+        for (v = 0; v < options.NVEGTYPES; v++) {
+            vidx = veg_con_map[i].vidx[v];
+            if (vidx != -1) {
+                for (j = 0; j < NF; j++) {
+                    if (veg_hist[i][vidx].vegcover[j] < MIN_VEGCOVER) {
+                        veg_hist[i][vidx].vegcover[j] = MIN_VEGCOVER;
+                    }
+                }
+            }
+        }
```

Comment:
You're right, it's a correction.  Sure, we could raise a warning.

The reason for this, by the way, is because we divide by LAI and albedo by vegcover and even if vegcover > 0, it could be so extremely small as to lead to large precision errors (although now that I think of it, I should make sure that this isn't done for tiles that are explicitly labeled as bare soil - I've run this code in regions with pure bare soil tiles, but what likely happened is that the bare soil tile was divided into a "vegetated" portion with LAI of 0 and a "bare" portion also with LAI of 0, resulting in the same end result but with twice the work... I can try to add a case to exclude this for pure bare soil tiles).